app/Child.php

```diff
@@ -137,4 +149,33 @@ public function family()
     {
         return $this->belongsTo('App\Family');
     }
+
+    /**
+     * Get info about whether child can defer
+     *
+     * @return boolean
+     */
+    public function getCanDeferAttribute()
+    {
+      $notices = $this->family->registrations[0]->getEvaluator()->evaluations["App\Child"]["notices"];
```

Comment:
1. This is quite a deep dependent call that assumes the Child class knows a lot about the arrangement of the Family and registrations (cf. "encapsulation"); 

amend this->getEvaluator() as above and then call it here. This method doesn't need  to be the one that does that work. better to `this->getEvaluator()` where that function would check for a family and grab it's evaluator if necessary. 

2. Then once we have a rules set, check that `->evaluations["App\Child"]["notices] actually exists, else you risk a PHP 500 error when it tries to pull a key off an array. 

could do it with `->evaluations["App\Child"]["notices"] ?? []` as i think we want an empty array if it _can't_ find notices that apply to a Child.